packages/core/popper/src/popper.ts

```diff
@@ -1,5 +1,5 @@
 import * as CSS from 'csstype';
-import { makeRect, getOppositeSide, getCollisions } from '@radix-ui/utils';
+import { makeRect, getOppositeSide, getCollisions, contractRect } from '@radix-ui/utils';
```

Comment:
Yeah it is contract as in expand/contract.

Reply:
Agreed on the naming here. Maybe `makeRectWithTolerance` since it's already composing `makeRect` and just accounting for tolerance?

Reply:
I see what you mean though, I couldn't find a better name…
The thing is this lives in some basic geometry utils, and tolerance doesn't really mean anything there.
It's really just taking a rect, and applying an offset on each side, ie. it's contracting it by a given amount on each side.

Reply:
I guess if we can't find a better name we could always import it like `contractRect as getRectWithTolerance`?

Reply:
`getRectWithOffset`?

Reply:
What is the issue though? I personally think that from the perspective of the geometry utility file, the name `contractRect` and signature `function contractRect(rect: ClientRect, amount: number)` is pretty self explanatory.

I would think that we should retain this.

However, I am happy to use a more specific name when using inside `popper.ts` as there's more context there as to what we're trying to do (ie. why I suggested the `import … as …` syntax)

Reply:
I disagree that it's self explanatory. To me, `contractRect` sounds like it takes a rect and mutates it in some way. It actually takes a rect, adjusts it by an offset and returns a new value. Also `contract` could also be `expand` depending on the value passed to `amount`.  🤷‍♂️ 

I don't think this is super important since it's internal anyway, but that's just my two cents on it. It was unclear to me at first glance.

Reply:
Right, I think the confusion was more to do with seeing this used in `popper.ts` I'd say. I don't think it would have been too much in question.

Happy to go with a general name that would work well in both, but I'm not sure what that is…

`getRechWithOffset` to me is confusing, it's sounds more about moving the rect that resizing it, although in theory it's doing both because it's shriking/expanding from center-point.

Reply:
`getContractedRect`?